CADETPythonSimulator/unit_operation.py

```diff
@@ -485,8 +513,8 @@ def compute_residual(
 
         """
         # Inlet DOFs are simply copied to the residual.
-
-        self.residuals['outlet']['c_poly'] = self.states['outlet']['c_poly']
+        t_poly = np.array([1, t, t**2, t**3])
```

Comment:
Do we still need this or is this now handled by Sections? Maybe we need to discuss this again. But could also be moved to a follow-up issue.

Reply:
Not sure what you mean. Maybe it needs to be discussed again.